packages/packages/commonwell-sdk/src/client/commonwell.ts

```diff
@@ -150,6 +206,136 @@ export class CommonWell {
+   * @param meta    Metadata about the request.
+   * @param patient  The patient to register.
+   * @returns
+   */
+  async registerPatient(meta: RequestMetadata, patient: Patient): Promise<Patient> {
+    const headers = await this.buildQueryHeaders(meta);
+    const resp = await this.api.post(`/org/${this.oid}/patient`, patient, {
+      headers,
+    });
+    return patientSchema.parse(resp.data);
+  }
+
+  /**
+   * Searches for a patient based on params.
+   * See: https://specification.commonwellalliance.org/services/patient-identity-and-linking/protocol-operations#8761-search-for-a-patient
+   *
+   * @param meta    Metadata about the request.
+   * @param fname   Patient's first name.
+   * @param lname   Patient's last name.
+   * @param dob     Patient's date of birth.
+   * @param gender  Patient's gender.
+   * @param zip     Patient's zip code.
+   * @returns
+   */
+  async searchPatient(
+    meta: RequestMetadata,
+    fname: string,
+    lname: string,
+    dob: string,
+    gender: string,
+    zip: string,
+  ): Promise<PatientSearchResp> {
+    const headers = await this.buildQueryHeaders(meta);
+    const resp = await this.api.get(`/org/${this.oid}/patient`, {
+      headers,
+      params: { fname, lname, dob, gender, zip },
+    });
+    return patientSearchRespSchema.parse(resp.data);
+  }
+
+  /**
+ * Updates a patient.
+ * See: https://specification.commonwellalliance.org/services/patient-identity-and-linking/protocol-operations#8763-updating-a-local-patient-record
+ *
+ * @param meta    Metadata about the request.
+ * @param patient  The data to update.
+ * @param id      The patient to be updated.
+ * @returns
+ */
+  async updatePatient(meta: RequestMetadata, patient: Patient, id: string): Promise<Patient> {
+    const headers = await this.buildQueryHeaders(meta);
+    const resp = await this.api.post(`/org/${this.oid}/patient/${id}/`, patient, {
+      headers,
+    });
+    return patientSchema.parse(resp.data);
+  }
+
+  /**
+* Merges patients.
+* See: https://specification.commonwellalliance.org/services/patient-identity-and-linking/protocol-operations#8765-merging-local-patient-records
+*
+* @param meta    Metadata about the request.
+* @param nonSurvivingPatientId  The local Patient Identifier of the non-surviving Patient Record (This patient gets replaced)
+* @param referencePatientLink   The patient link for the patient that will replace the non surviving patient
+* @returns
+*/
+  async mergePatients(meta: RequestMetadata, nonSurvivingPatientId: string, referencePatientLink: string): Promise<void> {
+    const headers = await this.buildQueryHeaders(meta);
+    await this.api.put(`/org/${this.oid}/patient/${nonSurvivingPatientId}/merge`,
+      {
+        "link":
+        {
+          "other": {
+            "reference": referencePatientLink
+          },
+          "type": "replace"
+        }
+      },
+      {
+        headers,
+      });
+    return;
+  }
+
+
+  // TODO: Not returning from other orgs
```

Comment:
In reference to my comment regarding network links

Reply:
See my comment above